ast/builtins.go

```diff
@@ -1709,17 +1709,21 @@ var JSONMarshal = &Builtin{
 
 var JSONMarshalIndent = &Builtin{
 	Name: "json.marshal_indent",
-	Description: "Serializes the input term to indented JSON, including line breaks. " +
+	Description: "Serializes the input term to indented JSON, including line breaks, with optional prefix. " +
 		"One or more copies of the indent string are emitted before each child JSON element, depending on indent level.",
 	Decl: types.NewFunction(
 		types.Args(
 			types.Named("x", types.A).Description("the term to serialize"),
 			types.Named("indent", types.NewAny(
 				types.S,
 				types.NewNull(),
-			)).Description("the string to indent with, defaults to `\\t` (tab character, 0x09) if null"),
+			)).Description("the string to indent with, defaults to `\\t` (horizontal tab character, 0x09) if null"),
```

Comment:
Interesting choice. I think I'd have defaulted to `"  "` (two spaces), but it's a matter of preference. We don't have a strong convention for "default" parameters anyways... I suppose I'd expect most people to just write what they want, not use `null` to pick some default.

Reply:
I normally prefer tab-indented JSON because it sidesteps the 2-vs-4-spaces debate (people can set their editors' tab width to whatever suits their fancy). They're also escaped inside of strings, unlike spaces.

Happy to change the default if the consensus is that some quantity of spaces would be significantly preferable.

Reply:
I like tabs for the very same reasons 👍. And we're providing users with an easy means to override the default.